### PR TITLE
PT-167, PT-168 | Edit event page

### PR DESCRIPTION
**Pull request by jorilindell**
1. Page to edit existing event
2. Support different language versions of the event
3. Show warning if there are unsaved changes on the form

### CODE REVIEW

File: src/domain/event/CreateEventPage.tsx [hunk 1]
```diff
@@ -2,85 +2,45 @@ import React from 'react';
...
-  // TODO: Get event language which user has selected when implemented
-  const language = locale;
 
   return (
     <PageWrapper title="createEvent.pageTitle">
       <Container>
         <div className={styles.eventPage}>
-          <h1>{t('createEvent.title')}</h1>
           <EventForm
             onSubmit={async (values) => {
```
Comment (Sartonon) [inline src/domain/event/CreateEventPage.tsx:27]: maybe move this logic to its own function from the jsx

File: src/domain/event/EditEventPage.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  return (
+    <PageWrapper title="editEvent.pageTitle">
+      <LoadingSpinner isLoading={loading}>
+        <Container>
+          <div className={styles.eventPage}>
+            <EventForm
+              eventData={eventData}
+              initialValues={initialValues}
+              onSubmit={async (values) => {
```
Comment (Sartonon) [inline src/domain/event/EditEventPage.tsx:72]: maybe move this to its own function

File: src/domain/event/eventDetailsButtons/EventDetailsButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+const EventDetailsButtons: React.FC<Props> = ({
+  eventData,
+  onClickLanguage,
+  selectedLanguage,
+}) => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const eventId = eventData.event?.id || '';
+
+  const moveToEventList = () => {
```
Comment (Sartonon) [inline src/domain/event/eventDetailsButtons/EventDetailsButtons.tsx:29]: would goToEventList be better name?

File: src/domain/event/eventDetailsButtons/EventDetailsButtons.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    history.push(ROUTES.HOME);
+  };
+
+  const moveToEditPage = () => {
```
Comment (Sartonon) [inline src/domain/event/eventDetailsButtons/EventDetailsButtons.tsx:33]: would goToEditPage be better name?

File: src/domain/event/utils.ts [hunk 5]
```diff
@@ -13,3 +18,70 @@ export const getEventPlaceholderImage = (id: string): string => {
...
   return EVENT_PLACEHOLDER_IMAGES[index];
 };
+
+/**
+ * Get first availanle language of an event
+ * @param {object} eventData
+ * @return {string}
+ */
+export const getFirstAvailableLanguage = (eventData: EventQuery): Language => {
+  if (eventData.event?.name.en) return SUPPORT_LANGUAGES.EN;
```
Comment (Sartonon) [inline src/domain/event/utils.ts:28]: Is this the correct order? should FI be first?

File: src/domain/event/utils.ts [hunk 6]
```diff
@@ -13,3 +18,70 @@ export const getEventPlaceholderImage = (id: string): string => {
 
   return EVENT_PLACEHOLDER_IMAGES[index];
 };
+
+/**
+ * Get first availanle language of an event
```
Comment (Sartonon) [inline src/domain/event/utils.ts:23]: btw why do we need these docs?

File: src/domain/event/utils.ts (hunk 5 above)
Comment (Sartonon) [reply src/domain/event/utils.ts:28]: maybe should be same order as in the language buttons?

Screenshot 2020-05-19 at 11 34 39

File: src/domain/event/utils.ts (hunk 6 above)
Comment (jorilindell) [reply src/domain/event/utils.ts:23]: Maybe we don't. I'm just used to write them

**Pull request merged**